### PR TITLE
Cleanup npm scripts

### PR DESCRIPTION
**Pull request by SimenB**
Somewhat shorter scripts, and a little DRYer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,10 +73,11 @@
     "web_modules/"
   ],
   "scripts": {
-    "test": "eslint lib && mocha --reporter spec",
```
Comment (SimenB) [inline package.json]: `spec` is default reporter

**Comment by sokra** [comment]
the coveralls documentation recommends `--report lcovonly`. Should we add a `cover-light` task that uses this?

**Comment by SimenB** [comment]
@sokra Hmm, `lcov` is produced in any case. What's the harm in producing some html and css as well? It's probably a bit faster, but hardly noticeable...

Anyway, I re-added the `travis` goal, so it's all good :smile:

**Comment by SimenB** [comment]
@sokra Ah, `--` is only supported by `npm` > 2.
What do you recommend? We can do `npm i -g npm` before running the test-script?

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,12 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
+#  - "iojs"
```
Comment (SimenB) [inline .travis.yml]: Seems to have problems with this...

**Comment by sokra** [comment]
omg... coverall spams a lot... ;)

**Comment by SimenB** [comment]
@sokra Haha, I know. Trying to get the stupid `iojs` build to go, so have been pushing a lot...

I'll just comment out `iojs` for now, and I've reported this https://github.com/wadey/node-microtime/issues/26
Might be more correct to report it at travis, maybe?

**Comment by sokra** [comment]
Thanks

**Pull request merged**